plonky2/src/hash/merkle_tree.rs

```diff
@@ -194,16 +214,45 @@ mod tests {
         const D: usize,
     >(
         leaves: Vec<Vec<F>>,
-        n: usize,
+        cap_height: usize,
```

Comment:
Changing the signature introduce `cap_height`. Got rid of `n` while I was at it because it was redundant.